aws/resource_aws_elasticache_replication_group.go

```diff
@@ -809,6 +809,20 @@ func resourceAwsElasticacheReplicationGroupUpdate(d *schema.ResourceData, meta i
 		}
 	}
 
+	if d.HasChange("auth_token") {
```

Comment:
does this mean if auth_token is provided, it will always try to rotate it? 
As I think if a user manually rotates the token, ie
1. execute ROTATE AUTH with newToken
2. execute SET AUTH with newToken 
and updates the newToken in terraform, terraform plan should not show any changes right?
Please correct me if I am missing anything, thanks

Reply:
No. It will only update if there is a change to `auth_token`.